packages/contentstack-clone/package.json

```diff
@@ -5,22 +5,22 @@
   "author": "Contentstack",
   "bugs": "https://github.com/rohitmishra209/cli-cm-clone/issues",
   "dependencies": {
+    "@colors/colors": "^1.5.0",
     "@contentstack/cli-cm-export": "~1.11.0",
     "@contentstack/cli-cm-import": "~1.14.0",
     "@contentstack/cli-command": "~1.2.16",
     "@contentstack/cli-utilities": "~1.5.13",
-    "@colors/colors": "^1.5.0",
     "async": "^3.2.4",
     "chalk": "^4.1.0",
     "child_process": "^1.0.2",
     "fancy-test": "^1.4.10",
     "inquirer": "8.2.4",
+    "lodash": "^4.17.20",
+    "merge": "^2.1.1",
     "ora": "^5.1.0",
     "prompt": "^1.3.0",
-    "rimraf": "^3.0.2",
-    "winston": "^3.7.2",
-    "merge": "^2.1.1",
-    "lodash": "^4.17.20"
+    "rimraf": "^5.0.5",
```

Comment:
@aman19K As we've updated the MAJOR version, I hope it doesn't break any existing functionality.

Reply:
@antonyagustine ,This library is only being used for the cleanup step, which I took care of in clone.js. I'll ask Aniket to execute e2e before merging the code.